allennlp/tests/data/token_indexers/pos_tag_indexer_test.py

```diff
@@ -59,3 +59,13 @@ def test_as_array_produces_token_sequence(self):
         indexer = PosTagIndexer()
         padded_tokens = indexer.pad_token_sequence({'key': [1, 2, 3, 4, 5]}, {'key': 10}, {})
         assert padded_tokens == {'key': [1, 2, 3, 4, 5, 0, 0, 0, 0, 0]}
+
+    def test_blank_pos_tag(self):
+        tokens = self.tokenizer.split_words("allennlp is awesome.")
+        indexer = PosTagIndexer()
+        counter = defaultdict(lambda: defaultdict(int))
+        for token in tokens:
+            indexer.count_vocab_items(token, counter)
+        vocab = Vocabulary(counter)
+        # should raise no exception
+        indexer.tokens_to_indices(tokens, vocab, index_name="pos")
```

Comment:
This one doesn't actually test a blank pos tag, does it?  Maybe add a blank pos tag, and check the result?